### PR TITLE
Squash error if .julia exists

### PR DESCRIPTION
**Pull request by asinghvi17**

### CODE REVIEW

File: src/conversions.jl [hunk 1]
```diff
@@ -479,6 +479,55 @@ function convert_arguments(
...
+    x, y = PlotUtils.adapted_grid(f, endpoints(i))
+    ptype = plottype(P, Lines)
+    to_plotspec(ptype, convert_arguments(ptype, x, y))
+end
+
+to_tuple(t::Tuple) = t
+to_tuple(t) = (t,)
+
+function convert_arguments(P::PlotFunc, f::Function, args...; kwargs...)
+    tmp = f(args...; kwargs...) |> to_tuple
```
Comment (SimonDanisch) [inline src/conversions.jl]: to_tuple should already exist somewhere, and please dont use `|>`

Comment (asinghvi17) [reply src/conversions.jl]: That's the only place it's defined, when I removed it and forgot to paste back I got an undefvar error for it.
```suggestion 
    tmp =to_tuple(f(args...; kwargs...))

Comment (asinghvi17) [reply src/conversions.jl]: We could relocate it somewhere else?

Comment (SimonDanisch) [reply src/conversions.jl]: Ah lol, you just moved it around - i guess I meant exactly that definition then :D well, then lets just remove the pipe :D

Comment (asinghvi17) [reply src/conversions.jl]: done :)

**Comment by SimonDanisch** [comment]
Did you try running the code that uses `tryrange` ?

**Comment by asinghvi17** [comment]
Haven't used it in anything yet, I think that got in by accident

**Pull request merged**